crates/holochain_chc/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "holochain_chc"
+version = "0.1.0-dev.1"
+description = "Defines the Chain Head Coordinator (CHC) API for Holochain and provides an HTTP client implementation."
+license = "Apache-2.0"
+homepage = "https://github.com/holochain/holochain"
+documentation = "https://docs.rs/holochain_chc"
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
+edition = "2021"
+
+# reminder - do not use workspace deps
+[dependencies]
+async-trait = "0.1"
+derive_more = "0.99"
+futures = "0.3"
+getrandom = "0.2.7"
+holochain_keystore = { version = "^0.4.0-dev.18", path = "../holochain_keystore", default-features = false }
+holochain_nonce = { version = "^0.4.0-dev.6", path = "../holochain_nonce" }
+holochain_types = { path = "../holochain_types", version = "^0.4.0-dev.20" }
+one_err = "0.0.8"
+must_future = "0.1.1"
+parking_lot = "0.12"
+serde = { version = "1.0", features = ["derive"] }
+serde_bytes = "0.11.12"
+serde_json = { version = "1.0.51", features = ["preserve_order"] }
+thiserror = "1.0.22"
+tracing = "0.1"
+url = "2.4"
+url2 = "0.0.6"
```

Comment:
Any specific reason for using both rather than one of these deps?

Reply:
removed url2